Karma/Controllers/ChatsController.cs

```diff
@@ -95,6 +126,18 @@ private Chat CreateChat(CreateChatModel chatData, Chat chat = null)
             return chat;
         }
 
+        private int UnseenMessagesCount(string userId)
+        {
+            return _context.Chat
```

Comment:
There seems to be a problem. I think you haven't added the latest migration you made?
Because I did run `update-database` when reviewing your branch.

![Screenshot 2021-11-14 at 13 05 24](https://user-images.githubusercontent.com/73688133/141678336-f7a5790e-f375-4a3a-8300-eaea964ce3dd.png)

Reply:
Nevermind, I didn't think of just running  `add-migration`